ssc/test/Test/Pos/Ssc/Arbitrary.hs

```diff
@@ -262,19 +261,3 @@ instance Arbitrary MCShares where
 instance Arbitrary MCVssCertificate where
     arbitrary = genericArbitrary
     shrink = genericShrink
-
-instance Arbitrary (DataMsg MCCommitment) where
-    arbitrary = genericArbitrary
-    shrink = genericShrink
-
-instance Arbitrary (DataMsg MCOpening) where
-    arbitrary = genericArbitrary
-    shrink = genericShrink
-
-instance Arbitrary (DataMsg MCShares) where
-    arbitrary = genericArbitrary
-    shrink = genericShrink
-
-instance Arbitrary (DataMsg MCVssCertificate) where
-    arbitrary = genericArbitrary
-    shrink = genericShrink
```

Comment:
Moved to [here](https://github.com/input-output-hk/cardano-sl/pull/3233/files#diff-e17f20f2dde34274e7596f910329d013R268)